Microsoft.Toolkit.Uwp.Services/Services/Bing/BingDataProvider.cs

```diff
@@ -41,7 +42,7 @@ public class BingDataProvider : DataProviderBase<BingSearchConfig, BingResult>
         protected override async Task<IEnumerable<TSchema>> GetDataAsync<TSchema>(BingSearchConfig config, int maxRecords, IParser<TSchema> parser)
         {
             var countryValue = config.Country.GetStringValue();
-            var locParameter = string.IsNullOrEmpty(countryValue) ? string.Empty : $"loc:{countryValue}+";
+            var locParameter = string.IsNullOrEmpty(countryValue) ? $"loc:{CultureInfo.CurrentCulture.Name.Split('-')[1].ToLower()}+" : $"loc:{countryValue}+";
```

Comment:
Corrent. There's no guarantee the dash is there. It depends on whether its a neutral language or not.
example:
```
            var c = new CultureInfo("en"); //Name returns "en" and IsNeutralLanguage is true
            var c2 = new CultureInfo("en-US"); //Name returns "en-US" and IsNeutralLanguage is false
```